packages/core-components/api-report.md

```diff
@@ -1471,5 +1488,6 @@ export type WarningPanelClassKey =
 // src/components/DependencyGraph/types.d.ts:26:9 - (ae-unresolved-link) The @link reference could not be resolved: The package "@backstage/core-components" does not have an export "DependencyNode"
 // src/components/TabbedLayout/RoutedTabs.d.ts:9:5 - (ae-forgotten-export) The symbol "SubRoute_2" needs to be exported by the entry point index.d.ts
 // src/components/Table/Table.d.ts:20:5 - (ae-forgotten-export) The symbol "SelectedFilters" needs to be exported by the entry point index.d.ts
-// src/layout/ErrorBoundary/ErrorBoundary.d.ts:8:5 - (ae-forgotten-export) The symbol "SlackChannel" needs to be exported by the entry point index.d.ts
+// src/layout/ErrorBoundary/ErrorBoundary.d.ts:13:5 - (ae-forgotten-export) The symbol "FallbackProps" needs to be exported by the entry point index.d.ts
```

Comment:
Could you name this `ErrorBoundaryFallbackProps` and export it too, to get rid of the warning?